scripts/python_codegen.sh

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env bash
+
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+set -e pipefail
```

Comment:
Reading [this page](https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425) it seems it could be useful to set other flags, such as `set -eEuo`.